build.gradle

```diff
@@ -1,7 +1,32 @@
+buildscript {
+    repositories {
+        mavenLocal()
+        maven {
+            name "GradlePlugins"
+            url 'https://plugins.gradle.org/m2/'
+        }
+        maven {
+            name "MavenSnapshot"
+            url 'https://oss.sonatype.org/content/repositories/snapshots/'
+        }
+    }
+
+    dependencies {
+        classpath group: 'com.diffplug.gradle.spotless', name: 'spotless', version: '2.4.1'
+        classpath group: 'org.junit.platform', name: 'junit-platform-gradle-plugin', version: '1.0.0'
+        classpath group: 'com.squareup.okhttp3', name: 'okhttp', version: '3.9.0'
+        classpath group: 'org.ajoberstar', name: 'grgit', version: '2.0.1'
+    }
+}
+
+plugins {
+    id 'com.github.ben-manes.versions' version '0.15.0'
```

Comment:
What do we need this for?

Reply:
Can get rid of this is we would like - it can be used manually like `./gradlew dependencyUpdates` to get a report of all out-of-date dependencies

Reply:
Oh, that's actually pretty nice. Leave it in then, and I will give it a go when I build this branch locally.